src/chainstate/burn/db/sortdb.rs

```diff
@@ -2843,10 +2848,17 @@ impl SortitionDB {
         match epoch {
             StacksEpochId::Epoch10 => true,
             StacksEpochId::Epoch20 => {
-                version == "1" || version == "2" || version == "3" || version == "4"
+                version == "1"
+                    || version == "2"
+                    || version == "3"
+                    || version == "4"
+                    || version == "5"
+            }
+            StacksEpochId::Epoch2_05 => {
+                version == "2" || version == "3" || version == "4" || version == "5"
             }
-            StacksEpochId::Epoch2_05 => version == "2" || version == "3" || version == "4",
-            StacksEpochId::Epoch21 => version == "3" || version == "4",
+            StacksEpochId::Epoch21 => version == "3" || version == "4" || version == "5",
```

Comment:
Just want to verify that Epoch21 supports schema v5? Assuming so since you truncate epocs table earlier.

Reply:
Yep, version 5 can run all epochs.